app/serializers/sprangular/base_serializer.rb

```diff
@@ -0,0 +1,7 @@
+require 'active_model_serializers'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.